src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -151,6 +151,33 @@ protected SemanticModel GetSemanticModel(SyntaxTree tree)
 
                 AddOverloadCountPart(symbols);
                 FixAllAnonymousTypes(symbols[0]);
+                AddExceptions(symbols[0]);
+            }
+
+            private void AddExceptions(ISymbol symbol)
+            {
+                // clean up the list of possible exceptions by de-duplicating and ordering them
+                var exceptions =
+                    symbol.GetDocumentationComment().ExceptionTypes
+                    .Distinct()
+                    .OrderBy(e => e)
```

Comment:
:exclamation: If these are strings, you need to specify `StringComparer.____IgnoreCase` (the `____` would be up to other reviewers).